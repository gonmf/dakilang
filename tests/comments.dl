# Some tests with comments
          # Bellow

mammal(A) :- # Not a perfect definition \
  blood(A, 'warm'), \
  (hasHair(A); hashFur(A)), \
  # laysEggs(A), Not all of them lay legs! \
  producesMilk(A).

listing
