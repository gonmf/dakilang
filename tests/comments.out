> # Some tests with comments
> # Bellow
>
> mammal(A) :- # Not a perfect definition    blood(A, 'warm'),    (hasHair(A); hashFur(A)),    # laysEggs(A), Not all of them lay legs!   producesMilk(A).
>
> listing
0: mammal(A) :- blood(A, 'warm'), hasHair(A), producesMilk(A).
1: mammal(A) :- blood(A, 'warm'), hashFur(A), producesMilk(A).

